adafruit_ticks.py

```diff
@@ -122,6 +122,8 @@ def ticks_ms() -> int:
 
 def ticks_add(ticks: int, delta: int) -> int:
     "Add a delta to a base number of ticks, performing wraparound at 2**29ms."
+    if delta <= -_TICKS_PERIOD / 2 or delta >= _TICKS_PERIOD / 2:
+        raise OverflowError("ticks interval overflow")
     return (ticks + delta) % _TICKS_PERIOD
```

Comment:
reuse existing half-period constant (which wasn't previously used, but was defined)
```suggestion
    if  -_TICKS_HALFPERIOD < delta < _TICKS_HALFPERIOD:
        return (ticks + delta) % _TICKS_PERIOD
     raise OverflowError("ticks interval overflow")
```

Check my boundaries so I'm not off by one on either side.